runtime/lua/vim/lsp/buf.lua

```diff
@@ -297,26 +297,30 @@ local function pick_call_hierarchy_item(call_hierarchy_items)
   return choice
 end
 
+local function call_hierarchy(method)
+  local params = util.make_position_params()
+  request('textDocument/prepareCallHierarchy', params, function(err_message, _, result)
+    if err_message then
+      vim.notify(err_message.message, vim.log.levels.ERROR)
```

Comment:
Should this be an error or info?

Reply:
I've changed it to WARN for now to be aligned with the formatting functions. Feel free to change them both later if you think it's needed. Either way, I think it should at least be WARN, as something went wrong from the caller's perspective.